src/test/java/org/example/BaseTests.java

```diff
@@ -43,24 +46,38 @@ public class BaseTests {
 	protected HomePage homePage;
 	protected WebDriverWait wait;
 	protected static final Logger logger = Logger.getLogger(BaseTests.class.getName());
+	private static final JiraService jiraService = new JiraService(
+        "https://your-jira-instance.atlassian.net",
+        "juan_ocampo",
+        "Jira_password01"
```

Comment:
Please, remove the credentials from the code. think how to provide this sensitive data through command line or some another way,

Reply:
Final private variables created